src/core/net/tcp6.cpp

```diff
@@ -95,40 +125,69 @@ const SockAddr &Tcp::Endpoint::GetPeerAddress(void) const
 
 Error Tcp::Endpoint::Bind(const SockAddr &aSockName)
 {
-    OT_UNUSED_VARIABLE(aSockName);
+    Error error;
+    VerifyOrExit(!static_cast<const Address *>(&aSockName.mAddress)->IsUnspecified(), error = kErrorInvalidArgs);
```

Comment:
Newline after vairable definitions and first statement?

Reply:
OK. Fixed in the latest update.